public/static/docs/command-reference/get.md

```diff
@@ -6,6 +6,10 @@ into the current working directory.
 > Unlike `dvc import`, this command does not track the downloaded files (does
 > not create a DVC-file).
 
+<!-- Separate MD quote: -->
+
+> See also our `dvc.api.open` Python API function.
```

Comment:
same, we are complicating too much the structure

Reply:
Same (0a725e8e)